faf-commons-lobby/src/main/kotlin/com/faforever/commons/lobby/SocialApi.kt

```diff
@@ -25,46 +29,71 @@ interface SocialApi {
 /**
  * Regular update on the players currently online
  */
-internal data class PlayerInfo(
+data class PlayerInfo(
```

Comment:
Why is this not internal anymore? It should ensure that nobody is reusing/coupling this class where it doesn't belong. Also applies to the others below

Reply:
These are server messages that would need to be acted on by whatever implements the lobby client by accessing them through the event flux. If these classes are kept internal then there is no visibility to interact with these classes or messages.

Unless there is some other way I should be acting on messages received through the event flux.